README.md

```diff
@@ -1,2 +1,88 @@
-# docker-cuda-tk1
-Docker development environment for the NVIDIA TK1
+# Docker-CUDA-AGX
+Docker development environment for the NVIDIA Jetson AGX
+- Compatible for development for the NVIDIA Jetson Orin NX
+
+Jsoncpp is included in the installation. Please refer to the following instructions for installing the Docker Engine and setting up the Docker CUDA AGX. 
+
+## Installing Docker Engine on Ubuntu
+
+For more information, visit the [dockerdocs](https://docs.docker.com/engine/install/ubuntu/).
```

Comment:
If the following information is taken from instructions in this link, let's say so - sorta like a citation.

Reply:
That is a good point. I rephrased it so it's not beating around the bush and just says that the instructions are summarized from and to visit the provided link for more information.